components/alerts/alert.css

```diff
@@ -16,7 +16,7 @@
     max-width: 24rem;
     border-radius: 3px;
     margin: 0.7rem;
-    padding: 0.7rem 0.7rem;
+    padding: 0.7rem;
```

Comment:
one suggestion here -  since all versions of alerts are having the same set of properties, can you create a base class in the name of just `alert` and put all the properties in that. 

Also, I see that you. have sent the root font size to 90% and that brings 1 rem to 14.4 px I guess. do you think it will make things easier if you set the root font size to 62.5 or 100?

Reply:
acknowledged. I have changed this now and yes the code looks cleaner